packages/components/package.json

```diff
@@ -43,7 +42,7 @@
 		"vite-plugin-dts": "2.3.0",
 		"vitest": "0.31.1"
 	},
-	"dependencies": {
+	"peerDependencies": {
 		"pinia": "2.1.1",
 		"vue": "3.3.4",
 		"vue-i18n": "9.2.2",
```

Comment:
Not sure if we can move `vue` and `vue-i18n` to peerDependencies 🤔 I thought there was a reason for at least `vue` needing to be in `dependencies`.

Reply:
might be outdated knowledge on my part tho 🤔

Reply:
I cannot think of any 🤔🤔

Reply:
(cc @nickrum 😇)

Reply:
I think it depends on the package manager you install Directus with and whether the package `@directus/components` depends on has `vue` and `vue-i18n` listed as dependencies. As long as those dependencies can be resolved by Node, it should be fine using any package manager. If not, it depends on whether the package manager installs peer dependencies by default if they are not present.

By the way, we had issues with different `vue` versions being used at the same time before. That's why `vue` and a couple of others were peer dependencies of `@directus/types` with no exact version specified for quite some time.